dashboard/modules/job/tests/test_job.py

```diff
@@ -1,23 +1,154 @@
 import os
 import sys
+import copy
 import time
 import logging
 import requests
+import tempfile
+import zipfile
+import shutil
 import traceback
 
 import ray
 from ray._private.utils import hex_to_binary
 from ray.new_dashboard.tests.conftest import *  # noqa
+from ray.new_dashboard.modules.job import job_consts
 from ray.test_utils import (
     format_web_url,
     wait_until_server_available,
+    wait_for_condition,
+    wait_until_succeeded_without_exception,
 )
 import pytest
 
-os.environ["RAY_USE_NEW_DASHBOARD"] = "1"
-
 logger = logging.getLogger(__name__)
 
+TEST_PYTHON_JOB = {
+    "language": job_consts.PYTHON,
+    "runtimeEnv": {
+        "workingDir": "{web_url}/test/file?path={path}"
+    },
+    "driverEntry": "simple_job",
+}
```

Comment:
Btw, can you please post an RFC issue with the proposed JSON spec for this API so we can decide on the exact arguments and formatting?

Reply:
Good suggestion. I will update the JSON spec to this RFC issue: https://github.com/ray-project/ray/issues/13395